src/main/java/step2/domain/LottoMatchingCount.java

```diff
@@ -0,0 +1,34 @@
+package step2.domain;
+
+import java.util.Arrays;
+
+public enum LottoMatchingCount {
+    TREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000),
+    DEFAULT(0, 0);
+
+    private  final int count;
```

Comment:
```suggestion
    private final int count;
```

일관된 코드 스타일을 유지해주세요.

Intellij 에 내장되어 있는 reformat 코드 권장합니다.
**Window 기준 Ctrl + Alt + L / OS X 기준 Option + Command + L**

Reply:
이런 기능이 있었네요 감사합니다~ 제PC 윈도우 ctrl + alt +shift +L 이네요!